packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -108,7 +108,10 @@ SidePanel.propTypes = {
 	onSelect: React.PropTypes.func,
 	onToggleDock: React.PropTypes.func,
 	docked: React.PropTypes.bool,
-	selected: React.PropTypes.string,
+	selected: React.PropTypes.shape({
```

Comment:
This is not accurate, for what I understand, the selected is the action that is selected.
What I suggest : 
* create a const actionPropType
* `selected : actionPropType` and `actions: PropTypes.arrayOf(actionPropTypes)`